test/fixtures/admin/fixture_mbaas.js

```diff
@@ -25,6 +25,6 @@ module.exports = nock('https://apps.feedhenry.com')
 .put('/api/v2/mbaases/1a2b', '*')
 .reply(200, envReplies.crud)
 .delete('/api/v2/mbaases/1a2b', '*')
-.reply(200, envReplies.crud)
+.reply(204)
```

Comment:
Why expect `204` here? The backend actually returns the deleted entity (not sure if this makes sense though)